x/oracle/keeper/single.go

```diff
@@ -60,7 +60,7 @@ func recacheAggregatorContext(ctx sdk.Context, agc *aggregator.AggregatorContext
 	totalPower := big.NewInt(0)
 	validatorPowers := make(map[string]*big.Int)
 	k.IterateBondedValidatorsByPower(ctx, func(_ int64, validator stakingtypes.ValidatorI) bool {
-		power := big.NewInt(validator.GetConsensusPower(validator.GetBondedTokens()))
+		power := big.NewInt(validator.GetConsensusPower(sdk.DefaultPowerReduction))
```

Comment:
See here an example of this call within the SDK.
https://github.com/evmos/cosmos-sdk/blob/v0.47.5-evmos.2/x/staking/keeper/invariants.go#L115

While the SDK technically uses `stakingKeeper.PowerReduction()`, it is not a parameter but hardcoded to `sdk.DefaultPowerReduction`.
https://github.com/ExocoreNetwork/exocore/pull/53/files#diff-9bcaf321191d4f6106e27485bdfbbaef716b4e0e4f648d8e5b81a87bc93b6c77R59

Reply:
Would it be better to rename DefaultPowerReduction to VirtualUSDDecimal? This way, the token converted from power could be considered as a virtual USD token.  I didn't find the meaning of `DefaultPowerReduction` in the system.